src/java/org/apache/cassandra/cql3/Operator.java

```diff
@@ -299,7 +455,7 @@ private Operator(int b)
      */
     public void writeTo(DataOutput output) throws IOException
```

Comment:
I would rename these to de/serialize

Reply:
If you look at where those methods are used in `RowFilter` everything is about read/write. Changing the naming would be weird there.

Reply:
Icwym. It hurts my eyes but ok let's be consistent.